enhancements/node-tuning/mixed-cpu-node-plugin.md

```diff
@@ -0,0 +1,353 @@
+---
+title: mixed-cpu-node-plugin
+authors:
+  - "@Tal-or"
+reviewers: 
+  - "@MarSik"
+  - "@yanirq"
+  - "@ffromani"
+  - "@jmencak"
+  - "@mrunalp"
+  - "@dhellmann"
+approvers: 
+  - "@mrunalp"
+  - "@dhellmann"
+api-approvers:
+  - "@mrunalp"
+  - "@dhellmann"
+creation-date: 2023-05-03
+last-updated: 2023-06-14
+tracking-link: 
+  - https://issues.redhat.com/browse/CNF-7603
+  - https://github.com/openshift/enhancements/pull/1421
+  - https://github.com/openshift-kni/mixed-cpu-node-plugin
+---
+
+# mixed-cpu-node-plugin
+
+
+## Summary
+
+Resources management (particularly CPUs) in Kubernetes/OpenShift is limited and not flexible enough to cover all of
+our customer use cases.
+This enhancement introduces a new node-plugin that operates at runtime-level
+for extending CPU resources management on top of Kubernetes and OpenShift platforms.
+With the existing CPU management design, a container can either request exclusive CPUs or shared CPUs,
+while with this new node plugin, it would be possible for container workload to request for both.
```

Comment:
How does this work relate to dynamic resource allocation? https://kubernetes.io/docs/concepts/scheduling-eviction/dynamic-resource-allocation/

Reply:
It's not. 
DRA is an API at Kubernetes level, while this plugin meant to work at the runtime-level.
Using DRA as an API for activating this feature, is something that  we might consider in the future, but this has to be investigated if it's possible.